src/main/java/com/opentable/db/postgres/embedded/PreparedDbProvider.java

```diff
@@ -33,69 +31,80 @@
 import com.google.common.base.Preconditions;
 import com.google.common.base.Throwables;
 import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableSet;
-import com.google.common.collect.Maps;
 import com.google.common.util.concurrent.ThreadFactoryBuilder;
 
 import org.apache.commons.lang3.RandomStringUtils;
+import org.postgresql.ds.PGSimpleDataSource;
 
-public class EmbeddedPostgreSQLController
+public class PreparedDbProvider
 {
     private static final String JDBC_FORMAT = "jdbc:postgresql://localhost:%d/%s";
 
     /**
      * Each database cluster's <code>template1</code> database has a unique set of schema
      * loaded so that the databases may be cloned.
      */
-    @GuardedBy("EmbeddedPostgreSQLController.class")
-    private static final Map<Entry<URI, Set<String>>, Cluster> CLUSTERS = Maps.newHashMap();
```

Comment:
Google should deprecate `Maps.newHashMap` now that we have the diamond operator.